spark/spark-spa/src/main/java/com/thoughtworks/go/spark/spa/UsersController.java

```diff
@@ -52,8 +56,12 @@ public void setupRoutes() {
     }
 
     public ModelAndView index(Request request, Response response) {
+        boolean isSecurityEnabled = this.securityService.isSecurityEnabled();
```

Comment:
This does not look right. I think we ought to be explicitly checking if the `<admin/>` tag is present. See if you can delegate to `SecurityConfig#noAdminsRequired` through the security service?

Reply:
Yes, this doesn't seem right. All users are considered admins even if Security is enabled but if no admins are defined. Check - https://github.com/gocd/gocd/blob/master/config/config-api/src/main/java/com/thoughtworks/go/config/SecurityConfig.java#L91